docs/Materials.md.html

```diff
@@ -522,6 +523,13 @@
 
 ![Figure [materialUnlit]: The unlit model is used to render debug information](images/screenshot_unlit.jpg)
 
+## Specular glossiness
+
+This alternative lighting model exists to comply with legacy standards.
+
+Similar to cloth, it encompasses all the parameters previously defined for the standard
+lit mode except for _metallic_ and _reflectance_, and adds a _sheenColor_ parameter.
```

Comment:
Can you update the `### Material fragment inputs` section too? `sheenColor` is listed as cloth only. Also add a table to describe the parameter, like this:

```
       Parameter      |      Definition
---------------------:|:---------------------
**sheenColor**        | Specular tint to create two-tone specular fabrics (defaults to $\sqrt{baseColor}$)

       Property       |   Type   |            Range         |           Note
---------------------:|:--------:|:------------------------:|:-------------------------
**sheenColor**        | float3   |  [0..1]                  | Linear RGB
```

Except in a way that makes sense :)

Reply:
Done